libcloud/common/exception.py

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Minor thing - can you please call the module `exceptions.py` (plural) so it's consistent with other files (types, etc.)?